packages/opentelemetry-exporter-collector-grpc/README.md

```diff
@@ -32,48 +35,70 @@ provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
 
 provider.register();
 
+Graceful.on("exit", async () => {
+  await provider.shutdown();
+});
 ```
 
 By default, plaintext connection is used. In order to use TLS in Node.js, provide `credentials` option like so:
 
 ```js
 const fs = require('fs');
+// Must be 'grpc', _not_ '@grpc/grpc-js'
```

Comment:
this is strange as grpc is deprecated (or will be deprecated within the next weeks).

Reply:
In fact, I think we are using grpc-js

Reply:
strange. when I tried using @grpc/grpc-js I got type check errors.

Reply:
weird, I can no longer repro this failure. I'll update.

Reply:
I do see this in `exporter-collector-grpc` package deps right now though.

```
"@opentelemetry/exporter-collector-grpc@^0.18.2":
  version "0.18.2"
  resolved "https://registry.yarnpkg.com/@opentelemetry/exporter-collector-grpc>
  integrity sha512-bEQbcqTlinMaj+/7rH98M9+POGHOpdgTkD6a9LOym6Qu6eMF5F+TBqaUQjfG>
  dependencies:
    "@grpc/proto-loader" "^0.5.4"
    "@opentelemetry/api" "^0.18.1"
    "@opentelemetry/core" "^0.18.2"
    "@opentelemetry/exporter-collector" "^0.18.2"
    "@opentelemetry/metrics" "^0.18.2"
    "@opentelemetry/resources" "^0.18.2"
    "@opentelemetry/tracing" "^0.18.2"
    grpc "^1.24.2"
```

Reply:
https://github.com/open-telemetry/opentelemetry-js/blob/c0b021be5782ebee6a23015c992dc496b105dad2/packages/opentelemetry-exporter-collector-grpc/package.json#L70

not released yet but we're using grpc-js now

Reply:
gotcha. I'll fix. :)

Reply:
ah, okay. in the last release, it needed to be `'grpc'` and not `'@grpc/grpc-js'` or else this would happen.

```
       2021-04-22T00:43:30.299Z   189 E [server] Unhandled promise rejection
       2021-04-22T00:43:30.340Z   189 E TypeError: Channel's second argument (credentials) must be a ChannelCredentials
       2021-04-22T00:43:30.341Z   189 E     at ServiceClient.Client (/app/node_modules/grpc/src/client.js:460:23)
       2021-04-22T00:43:30.341Z   189 E     at new ServiceClient (/app/node_modules/grpc/src/client.js:986:12)
       2021-04-22T00:43:30.341Z   189 E     at /app/node_modules/@opentelemetry/exporter-collector-grpc/src/util.ts:54:35
```